frontend/src/pages/modelCatalog/screens/ModelDetailsView.tsx

```diff
@@ -33,7 +33,9 @@ const ModelDetailsView: React.FC<ModelDetailsViewProps> = ({ model }) => (
           <h2>Model card</h2>
           {!model.readme && <p className={text.textColorDisabled}>No model card</p>}
         </Content>
-        {model.readme && <MarkdownView data-testid="model-card-markdown" markdown={model.readme} />}
+        {model.readme && (
+          <MarkdownView data-testid="model-card-markdown" markdown={model.readme} maxHeading={3} />
```

Comment:
Nit - max/min-based naming like `maxHeading` is potentially confusing here because maxHeading=3 numerically implies that the allowed headings will be 1-3 when it really means 3-6.. but I'm not sure what a better name would be. `startHeadingsAt` or `initialHeading` or something maybe? max is fine, I'm overthinking it

Reply:
idk, i guess i'm open to change but to me h1 is the top heading so would be at the top and whatever is closest to that would be 'max', `startHeadingsAt` or `initialHeading` would be confusing to me.

maybe something like `maxHeadingSize` or something, but i doubt this will be used by anything else